### PR TITLE
Quarterly QA LWMS ID 221

### PR DESCRIPTION
**Pull request by vltabacaru**
# Add here your WMS ID 221

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

File: oracle-db-features-for-developers/intro/intro.md
Comment (denisemyrick) [inline oracle-db-features-for-developers/intro/intro.md:1]: remove references to Always Free and/or Free Tier

File: oracle-db-features-for-developers/json/json.md
Comment (denisemyrick) [inline oracle-db-features-for-developers/json/json.md:1]: Proceed to the next lab should be bolded

**Review by denisemyrick — DISMISSED** [review]
This branch is out of date with the base branch. Please merge the latest changes from main into this branch so it can be approved. Thanks!

**Comment by vltabacaru** [comment]
Changes have been done. Thanks

**Review by anooshapilli — COMMENTED** [review]
@vltabacaru - For the images that have missing alternate text, please check and add descriptive alternate text within the [] in the workshop markdown files
image

**Comment by vltabacaru** [comment]
Descriptive alternate text added.

**Review by klazarz — APPROVED** [review]

**Pull request merged**